src/modules/directory.rs

```diff
@@ -129,6 +109,38 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
     Some(module)
 }
 
+fn is_truncated(path: &str) -> bool {
```

Comment:
I think it would be better if the `path` was `&Path`

Reply:
I agree, but that module is pretty hairy.  The part that does path substitutions (aliases?) all works with strings, so it gets converted to strings somewhere else early on.  By the time it gets to my addition, it's already a string.

I tried to refactor it to always use &Path or &PathBuf, but it ended up a complete rewrite (due to the current version making it a string quite early on), so instead I went with a patch that changed as little as possible.